dracut/30afterburn/afterburn-network-kargs.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Afterburn initrd setup network kernel arguments
```

Comment:
Minor: I think the convention is usually Title Case for systemd unit descriptions.